js/background.js

```diff
@@ -39,7 +39,7 @@ function Background() {
       for(var i = 0; i < savedTabs.length; i++){ 
           var tab = savedTabs[i];
           if(tab.url){
-            tabs[tab.id] = {'trackers': {}, "total": 0, 'url': tab.url};
+            tabs[tab.id] = {'trackers': {}, "total": 0, 'url': tab.url, 'dispTotal': 0};
```

Comment:
> is the issue here that dispTotal is normally added, but isn't in some cases, so you're adding the default case?

Yes. Having a tabs module to create these for us would be a good improvement. 

> it's not necessary to quote the keys here, but if you do let's at least be consistent with which quote style we're using

done